src/Mod/Sketcher/Gui/CommandCreateGeo.cpp

```diff
@@ -267,24 +301,57 @@ class DrawSketchHandlerLine: public DrawSketchHandler
     {
         if (Mode==STATUS_SEEK_First) {
             setPositionText(onSketchPos);
+
             if (seekAutoConstraint(sugConstr1, onSketchPos, Base::Vector2d(0.f,0.f))) {
                 renderSuggestConstraintsCursor(sugConstr1);
                 return;
             }
+            if (toolWidgetManager.isParameterSet(0) + toolWidgetManager.isParameterSet(1) == 2) {

```

Comment:
This gave me a heart attack.

You pass a magic number to a function that returns a boolean. You pass a second magic number to a second function that returns a boolean. Then you implicitly convert booleans to integers for addition and compare it against another magic number (2).

Is there any reason why not to use &&, like in toolWidgetManager.isParameterSet(0) && toolWidgetManager.isParameterSet(1) ?

Do not change at this moment.

Reply:
No reason at all. Your way looks better :)